### PR TITLE
language privileges grid extension init

### PR DESCRIPTION
**Pull request by wiewiurdp**

### CODE REVIEW

File: module/product/src/Infrastructure/Grid/Builder/ProductGridColumnBuilder.php [hunk 1]
```diff
@@ -70,6 +86,9 @@ public function build(GridConfigurationInterface $configuration, Language $defau
     {
         $codes = $this->attributeQuery->getAllAttributeCodes();
 
+        /** @var User $user */
+        $user = $this->security->getUser();
```
Comment (rprzedzik) [inline module/product/src/Infrastructure/Grid/Builder/ProductGridColumnBuilder.php:90]: Security getUser can return null. 
need to handle this

File: module/product/src/Infrastructure/Grid/Builder/ProductGridColumnBuilder.php [hunk 2]
```diff
@@ -93,7 +111,9 @@ public function build(GridConfigurationInterface $configuration, Language $defau
                 $code = $column->getColumn();
                 $key = $column->getKey();
                 $language = $column->getLanguage() ?: $defaultLanguage;
-                if (in_array($code, $codes, true)) {
+
+                if (in_array($code, $codes, true)
+                    && $this->languagePrivilegesService->isReadableAccessGranted($languagePrivilegesCollection, $language)) {
```
Comment (rprzedzik) [inline module/product/src/Infrastructure/Grid/Builder/ProductGridColumnBuilder.php:116]: This should be implemented as User Aggregate Logic not in separate service

File: module/fixture/src/Infrastructure/Faker/LanguagePrivilegesFaker.php [hunk 3]
```diff
@@ -16,19 +16,615 @@
  */
 class LanguagePrivilegesFaker extends BaseProvider
 {
+    private const ALL_LANGAUGE_CODES = [
```
[the remark this answers is not in the document]
Comment (rprzedzik) [reply module/fixture/src/Infrastructure/Faker/LanguagePrivilegesFaker.php:19]: see Ergonode\Migration\Version20180610093112

[the remark this answers is not in the document]
Comment (wiewiurdp) [reply module/fixture/src/Infrastructure/Faker/LanguagePrivilegesFaker.php:19]: Initially I thought it is not needed but after discussions with Piotr, we agreed that validation needs to be added.

[the remark this answers is not in the document]
Comment (wiewiurdp) [reply module/fixture/src/Infrastructure/Faker/LanguagePrivilegesFaker.php:19]: about 'xx_XX' notation, I checked  Ergonode\Migration\Version20180610093112 and I also found there only 'xx' notation. Unless I read it wrong.

**Pull request merged**